.github/workflows/test.yml

```diff
@@ -67,12 +69,19 @@ jobs:
           VTK_VERSION=$(cat pyproject.toml | grep 'vtk==' | sed 's/vtk==//g' | sed 's/\"//g' )
           uv pip install --extra-index-url https://wheels.vtk.org vtk-osmesa==$VTK_VERSION
           uv pip install pytest-xdist pytest-cov
-      - name: Run tests
-        run: |
-          export C_INCLUDE_PATH=$pythonLocation/include/python${{ steps.python-selector.outputs.python-version }}:$C_INCLUDE_PATH
           scil_data_download
           scil_search_keywords test --regenerate_help_files --processes 8
-          uv run --active pytest --cov-report term-missing:skip-covered --dist=loadgroup
+
+      - name: Run unit tests
+        run: |
+          uv run --active pytest src/scilpy --cov-report term-missing:skip-covered --dist=loadgroup --ignore=src/scilpy/cli
+          mv .test_reports unit_test_reports
```

Comment:
Maybe the reason that smoke tests have the total coverage is that here you don't do `mv .coverage .coverage.unit`?

Reply:
Actually, the coverage report is also written in the `.test_reports` folder, so we only need to upload its content to coverage and it's sufficient.

I'm looking into the reports right now to find why we get those new numbers